### ISSUE TITLE
How are .weights files made?

### ISSUE THREAD

**Issue by BlrFox** [pjreddie/darknet#2493]
All info I could find was 'Golden Orchard Apple II, unknown file type' So I was just wondering what tools would someone need to make Rnn.weight files.

**Comment by Sujay-Mhaske** [comment]
By training

**Comment by BlrFox** [comment]
Correct, but Darknet it needs input "the .weight files" to _generate texts based on or from the training...._

**Comment by kurthorvath** [comment]
Hi,
this script did actually help me in my understanding how to do the training and how i use my own network for detection

https://colab.research.google.com/github/luxonis/depthai-ml-training/blob/master/colab-notebooks/YoloV3_V4_tiny_training.ipynb

i am not an expert here but i hope i did help
